.github/actions/setup/action.yml

```diff
@@ -1,30 +0,0 @@
-name: Test setup
-inputs:
-  pull_token:
-    description: "Token to use for private repo access"
-    required: true
-runs:
-  using: "composite"
-  steps:
-    - name: Set up git private repo access
```

Comment:
Are we sure we want to remove this? I thought the idea here was to avoid some of the github rate limiting issues we run into when installing the athena toolchain

Reply:
This is not used. The PULL_TOKEN is not even configured in the repo settings and if it was, it would only be passed in the release workflow (not in pr). I suspect it was needed in the repo it was copied from for pulling from private repos (and ours are public).